mlflow/resource_model.go

```diff
@@ -0,0 +1,126 @@
+package mlflow
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/databrickslabs/terraform-provider-databricks/common"
+	"github.com/hashicorp/terraform-plugin-sdk/v2/helper/schema"
+)
+
+// Model defines the parameters that can be set in the resource.
+type Model struct {
+	Name        string `json:"name" tf:"force_new"`
+	Tags        []Tag  `json:"tags,omitempty" tf:"force_new"`
+	Description string `json:"description,omitempty"`
+}
+
+// ModelDto defines the response object from the API
+type ModelDto struct {
+	Name                 string   `json:"name"`
+	CreationTimestamp    int64    `json:"creation_timestamp,omitempty" tf:"computed"`
+	LastUpdatedTimestamp int64    `json:"last_updated_timestamp,omitempty" tf:"computed"`
+	UserID               string   `json:"user_id,omitempty" tf:"computed"`
+	LatestVersions       []string `json:"latest_versions,omitempty" tf:"computed"`
+	Description          string   `json:"description,omitempty"`
+	Tags                 []Tag    `json:"tags,omitempty"`
+}
+
+// RegisteredModel defines response from GET API op
+type RegisteredModelDto struct {
```

Comment:
rename it to `registeredModel`, as it should be package private

Reply:
done